e2e/cypress/integration/api/iam/iam_integration.spec.ts

```diff
@@ -133,7 +133,7 @@ describe('assigning projects', () => {
 
     it(`when creating ${iamResources}, successfully assigns an existing authorized project`, () => {
       cy.request({
-        headers: { 'api-token': twoAllowedProjectsTok },
+        headers: { 'api-token': twoAllowedProjectsTok, 'content-type': 'application/json+lax' },
```

Comment:
do these tests fail now without the `content-type` defined?

Reply:
Unfortunately, yes. Everywhere the content-type is added is a tiny piece of tech-debt, I'm afraid.

I believe here, we're sending too many fields on purpose, to keep the code reusable for different `iamResources`? But I haven't dug too deep.